Repository/Vcs/GitDriver.php

```diff
@@ -35,4 +36,24 @@ public function getComposerInformation($identifier)
 
         return ProcessUtil::getComposerInformation($this->cache, $this->infoCache, $this->repoConfig['asset-type'], $this->process, $identifier, $resource, sprintf('git show %s', $resource), sprintf('git log -1 --format=%%at %s', escapeshellarg($identifier)), $this->repoDir, '@');
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function initialize()
+    {
+        if (isset($this->config->get('fxp-asset')['git-driver']['skip-update'])) {
```

Comment:
Ok I see, the using of the Git driver from the Github driver doesn't include this variable. I add that.